### ISSUE TITLE
[ErDiagram] Allow comments after entity name

### ISSUE THREAD

**Issue by xperiandri** [mermaid-js/mermaid#2548]
**Is your feature request related to a problem? Please describe.**
Usually, there is a need to describe the entity. So the ability to write a comment after an entity name would be extremely helpful! 

**Describe the solution you'd like**
Add ability to specify a comment using `''` quotes after an entity name at the end of the line or before the `{`

```
InitialSeller `(Company)` {
    ... 'attributes'
}
```

**Comment by RonaldZielaznicki** [comment]
@xperiandri 

The documentation is using single quotes while the [jison](https://github.com/mermaid-js/mermaid/blob/e6fc27886125137c929bbba2ae151a941b560811/src/diagrams/er/parser/erDiagram.jison#L23) file asks for double quotes.

You can see the proper use in the [spec](https://github.com/mermaid-js/mermaid/blob/e6fc27886125137c929bbba2ae151a941b560811/src/diagrams/er/parser/erDiagram.spec.js#L56) file too.

Looks like when I originally [added](https://github.com/mermaid-js/mermaid/pull/2237) the example image and code I didn't add in the example code, which was added in [here](https://github.com/mermaid-js/mermaid/pull/2420). 

And, I'm not remembering *why* I even used single quotes in the docs there. I think it had something to do with double quotes not behaving right? It's fuzzy. Either way, the fix for this would be to update the doc to use double quotes.

Going to ping @ddemoss222 since their [ping](https://github.com/mermaid-js/mermaid/pull/2237) is what pulled me over to look.

Oddly, even when using the right quotes, or just using PK/FK the ER diagram looks odd to me on the live editor. Which definitely wasn't how it looked when I originally pulled it together.

<img width="179" alt="image" src="https://user-images.githubusercontent.com/58531251/147532196-945e573c-03dc-4b73-9f59-ba3ab78ca3df.png">

[Related Issue](https://github.com/mermaid-js/mermaid/issues/2346)

**Comment by ddemoss222** [comment]
@RonaldZielaznicki Yes, this is how it is rendering for me as well. It looks as though it renders the text on top of each other. From some of the comments I reviewed within the issues it seemed as though the feature was supposed to add a column to the rendering for PK/FK, but I never saw a mock for how the comments were meant to render. Thanks for looking into this

**Comment by RonaldZielaznicki** [comment]
Looking a bit deeper, and at the second issue that I found on the live-editor side of things.

It looks like the error comes down to some math that wasn't done right [here](https://github.com/RonaldZielaznicki/mermaid/blob/0d91eee5e0c45369eff0aa3a0ba1e67cf6af411f/src/diagrams/er/erRenderer.js#L227) and [here](https://github.com/RonaldZielaznicki/mermaid/blob/0d91eee5e0c45369eff0aa3a0ba1e67cf6af411f/src/diagrams/er/erRenderer.js#L247).

Both use `typeRect` to set their own x position. Which isn't right. That's just the first node in the box. The one that holds the type. Really, the x position should be added to after each node.

So there are two things needing to be done to fix what's going on.

First is the single quotes in the docs. The second is this math that isn't showing right. I'm currently not on a computer that can easily do development work. Those are where I'm seeing the problems.


Still confounds me, because I rather vividly remember using this code to make my own diagrams at the time for a project of my own I was working on at the time. Hence why I did the change in the first place. Bah, guess hindsight is 20/20 and memory is short. If no one fixes these issues by Wednesday I'll nab them on Thursday.

**Comment by Arakade** [comment]
Just came to say the comments aren't rendering and they're also broken on the project's website:

https://mermaid-js.github.io/mermaid/#/entityRelationshipDiagram?id=attribute-keys-and-comments

Glad to see you're already on this!  Thanks 👍

**Comment by RonaldZielaznicki** [comment]
It looks like this issue was fixed by @ebjornset in January with #2598. Many thanks for grabbing it!

Small snag, it renders correctly in the live editor.

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/58531251/158064572-82199922-e027-4c41-b558-f3b0a0751ef5.png">

But, it doesn't look to be doing the same in the [docs](https://mermaid-js.github.io/mermaid/#/entityRelationshipDiagram?id=attribute-keys-and-comments)

<img width="975" alt="image" src="https://user-images.githubusercontent.com/58531251/158064601-2d49ab27-da8e-45a5-b5bb-6befd323c684.png">

The docs are using 8.13.0, and the fix went through in release 8.13.9. That would explain why it's happening, but it's not a good look for the project and could cause confusion. Let's get that updated.

**Comment by xperiandri** [comment]
My issue is about describing the whole entity, not its attributes.
Even if it is not visible. Just for documentation and explanation purposes.

**Comment by stumash** [comment]
you should force github to update their mermaid version, it doesn't render correctly on github becuase they are probably pre-8.14
